### ISSUE TITLE
Correct logic for related items display template

### ISSUE THREAD

**Issue by Guite** [Guite/MostGenerator#676]
Since entities contain their own actions in the 0.7.0 DSL, the generator can not rely on that all actions are available for all entities anymore.
This needs to be fixed for deciding whether embedded related items on a display page are supported or not.
If one A entity has a display action and many related B entities whereby B has no display action, current the following error occurs on A's display page:
```
WARN (4): E_USER_WARNING: Smarty error: unable to read resource: "B/include_displayItemListMany.tpl"
```

**Comment by Guite** [comment]
Check also the `RKGarage` model from @rallek

`Image/include_displayItemListMany.tpl` is not generated although the `Image` entity contains a display action.

**Comment by Guite** [comment]
Fixed in de94f586a033f215e8491db2010a5356ed502161
